### DISCUSSION TITLE
Add conditional page shadows

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#12789] [Feature Requests]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

When pages with gray extend beneath the sidebars, the gray on gray causes some visual confusion. We should add conditional shadows (same as header) for the left, right, and top.

### To Reproduce

Go to a Map Layout...
![Screen Shot 2021-09-27 at 7 08 43 PM](https://user-images.githubusercontent.com/522079/134997166-6a385fde-c395-4e48-ab86-e5867c7e2edf.png)

### What version of Directus are you using?

RC-95

### What version of Node.js are you using?

Node: 16.6.1

### What database are you using?

Postgres 13

### What browser are you using?

Chrome

### What operating system are you using?

macOS

### How are you deploying Directus?

DO

**Comment by licitdev** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
